Checkbox/Checkbox.module.less

```diff
@@ -0,0 +1,35 @@
+// Checkbox.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.toggleIcon {
+	.applySkins({
+		@size: @agate-checkbox-size;
+
+		width: @size;
```

Comment:
This might be too restrictive for agate. I'd be a bit surprised if a skin wanted a checkbox that wasn't the same size in both directions but they could. I'm okay to leave this but wanted to note for future consideration.